### PR TITLE
Simplify local setup instruction

### PR DESCRIPTION
**Pull request by louischan-oursky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,9 +131,94 @@ To run graphql server
...
+
+3. Configure authgear.yaml
+
+We need the following `authgear.yaml` to setup authgear for the portal.
+
+```yaml
+id: accounts # Make sure the ID matches AUTHGEAR_APP_ID environment variable.
+http:
+  # Make sure this matches the host used to access main Authgear server.
+  public_origin: 'http://localhost:3000'
```
Comment (carmenlau) [inline README.md:201]: Should it be `http://accounts.portal.localhost:3000`?

File: README.md [hunk 2]
```diff
@@ -129,9 +131,94 @@ To run graphql server
...
+oauth:
+  clients:
+    # Create a client for the portal.
+    # Since we assume the cookie is shared, there is no grant nor response.
+    - name: Portal
+      client_id: portal
+      # Note that the trailing slash is very important here
+      # URIs are compared byte by byte.
+      redirect_uris:
+        - "http://localhost:8000/oauth-redirect"
```
Comment (carmenlau) [inline README.md:216]: `http://portal.localhost:8000/oauth-redirect`?

File: README.md [hunk 3]
```diff
@@ -129,9 +131,94 @@ To run graphql server
...
+      post_logout_redirect_uris:
+        - "http://localhost:8000/oauth-redirect"
```
Comment (carmenlau) [inline README.md:218]: `http://portal.localhost:8000/oauth-redirect`?

**Comment by louischan-oursky** [comment]
Added a commit to clarify various redirect URIs.

**Pull request merged**